src/helpers/util.js

```diff
@@ -0,0 +1,5 @@
+export const logger = console;
```

Comment:
why are you storing the `console` global in a variable?

Reply:
Thank you for the question. eslint flags using console.log on any file. So, I saved it as variable and utilised it on server file, hence stopping the flag.

Reply:
If it's been decided to not have `console.log` in the codebase, the question would be why was that decision made? Is it useful? If it is, why try to bypass the tool you use to ensure compliance.

Someone can choose to something like.
`beans = console` and now we'll have `beans` all over the app. Or more realistically, everyone uses this `logger` all over the place. What then would have been the point of adding a `no-console` rule?

Think about this and possible solutions

Reply:
Thank you for this wonderful insight I would remove that now, but would it be okay to change it to consola ?

Reply:
The options available are node's `process.stdout.write()`, `console.log` or a logger like `morgan`

Reply:
I don't think this justifies adding a new dependency.

Reply:
Thank you for your feedback, I would implement process.stdout.write() now